p2panda-js/package.json

```diff
@@ -5,22 +5,18 @@
   "main": "lib/index.js",
   "browser": "lib/index.min.js",
   "scripts": {
-    "build": "run-s clear wasm webpack",
-    "clear": "rimraf ./lib ./wasm-web ./wasm-node",
+    "build": "run-s clear webpack",
+    "clear": "rimraf ./lib ./wasm",
     "lint": "eslint --ext .ts .",
     "test": "run-p --race test:mock-node test:run",
     "test:jest": "jest",
-    "test:run": "run-s clear wasm:node test:jest",
-    "test:watch": "nodemon --watch '../p2panda-rs/src/*' --watch './src/*' --exec 'npm test' --ext js,ts,json",
-    "test:template-openrpc": "npx ts-node ./test/generate-openrpc-spec.ts",
     "test:mock-node": "open-rpc-mock-server --port 2020",
-    "wasm": "run-s --silent wasm:browser wasm:node",
-    "wasm:browser": "wasm-pack --log-level warn build ../p2panda-rs --target web --out-dir ../p2panda-js/wasm-web --release --out-name index",
-    "wasm:node": "wasm-pack --log-level warn build ../p2panda-rs --target nodejs --out-dir ../p2panda-js/wasm-node --release --out-name index",
-    "wasm:copy": "mkdir -p ./lib/wasm; cp ./wasm-node/index.js ./lib/wasm; cp ./wasm-node/index.d.ts ./lib/wasm; cp ./wasm-node/index_bg.wasm ./lib/wasm",
-    "webpack": "run-p --aggregate-output webpack:browser webpack:node",
-    "webpack:browser": "webpack --mode production --config-name browser --color",
-    "webpack:node": "npm run wasm:copy && webpack --mode production --config-name node --color"
+    "test:run": "run-s clear webpack:node test:jest",
+    "test:template-openrpc": "npx ts-node ./test/generate-openrpc-spec.ts",
+    "test:watch": "nodemon --watch '../p2panda-rs/src/*' --watch './src/*' --exec 'npm test' --ext js,ts,json",
+    "webpack": "run-s webpack:web webpack:node",
+    "webpack:web": "webpack --mode production --config-name web --color",
+    "webpack:node": "webpack --mode production --config-name node --color"
```

Comment:
I think it may be helpful to allow running webpack in `development` mode. What do you think about setting the mode with `cross-env NODE_ENV=development` in the `test:run` script instead of hardcoding it here? The `build` script will then implicitly use `production` mode anyway. @adzialocha